base/docs/helpdb/Base.jl

```diff
@@ -786,6 +859,48 @@ at the position where it would appear if the array were fully sorted via a non-s
 algorithm. If `k` is a single index, that value is returned; if `k` is a range, an array of
 values at those indices is returned. Note that `select!` does not fully sort the input
 array.
+
+# Examples
+
```

Comment:
we should pick and standardize - personally I lean towards leaving out the empty line, assuming it should render the same